apps/marketing/src/app/(marketing)/open/monthly-new-users-chart.tsx

```diff
@@ -32,6 +32,9 @@ export const MonthlyNewUsersChart = ({ className, data }: MonthlyNewUsersChartPr
             <YAxis />
 
             <Tooltip
+              labelStyle={{
+                color: 'hsl(var(--primary-foreground))',
+              }}
```

Comment:
Why was this changed as part of this PR?

Reply:
The label colors weren't visible currently, so updated it to match the others

[Typefully.webm](https://github.com/documenso/documenso/assets/23498248/2ea2de1a-695d-49f3-8b0e-302d4d18dc6f)

Reply:
After the label changes, they look more like this, 

<img width="486" alt="Screenshot 2024-03-05 at 15 31 19" src="https://github.com/documenso/documenso/assets/23498248/0301679b-0922-4be2-a60c-49326dff77cf">